extensions/formats/gpx/src/main/java/mil/nga/giat/geowave/format/gpx/GpxIngestPlugin.java

```diff
@@ -313,4 +317,9 @@ public IngestGpxTrackFromHdfs(
 			Time.class
 		};
 	}
+
+	public void setExtentOptionProvider(
```

Comment:
is this called anywhere? Just trying to figure out where/how the JCommander annotations for this option provider get wired in.

Reply:
This is called from the GpxIngestFormat. I'm not really a fan of how I had to create the OptionProvider within the plugin + format. Ultimately, the JCommander annotations get set via SimpleFeatureIngestOptions because the Gpx specifc  plugin options get added via the internalGetIngestFormatOptionProviders. SimpleFeatureIngestOptions has the parameter delegate annotation set here. I attempted to attach it via GpxIngestPlugin originally, but since the plugin is created/returned as a AbstractSimpleFeatureIngestPlugin it wasn't picking up that annotation in the GpxIngestPlugin.
@ParametersDelegate
private Object pluginOptions = null;